htpasswd

```diff
@@ -0,0 +1 @@
+admin:$apr1$LgmO7/31$Hs7PTPLNCEKHnkcp4Lqbi/
```

Comment:
We should be careful here.  I doubt Seattle has changed the password, and I'm pretty sure this is the same password in our test deployment.  We should probably remove this file from version control, and add instruction in the documentation on how to set this up.